Products/ZenUI3/browser/resources/js/zenoss/form/DataPointItemSelector.js

```diff
@@ -13,6 +13,19 @@
 Ext.define("Zenoss.form.DataPointItemSelector", {
     alias:['widget.datapointitemselector'],
     extend:"Ext.ux.form.ItemSelector",
+    listeners: {
+        // show datapoint name and its RPN according to datapoints that
+        // were selected in selector form
+        change: function(obj, event) {
+            rpnTextObj = Ext.ComponentQuery.query('[name=rpnused]')[0];
+            if (rpnTextObj) {
+                rpnTextObj.setValue('');
+                Ext.each(obj.getValue(), function(dpname, index){
+                    rpnTextObj.setValue(rpnTextObj.value + dpname + ": " + rpnTextObj.record.allDataPoints[dpname] + "; ");
+                });
+            }
+        }
+    },
```

Comment:
When there are multiple datapoints, the presentation of datapoint and its RPN becomes difficult to read.  It'll be easy for users to misread or completely overlook a relevant RPN.  Having a (read-only) listview (i.e. a table) of the datapoint name and corresponding RPN would be an easy-to-read way to present this information to the user.